src/ape/managers/compilers.py

```diff
@@ -81,7 +81,10 @@ def compile(self, contract_filepaths: List[Path]) -> Dict[str, ContractType]:
             for path in paths_to_compile:
                 logger.info(f"Compiling '{self._get_contract_path(path)}'.")
 
-            compiled_contracts = self.registered_compilers[extension].compile(paths_to_compile)
+            base_path = self.config.PROJECT_FOLDER / Path("contracts")
```

Comment:
If we make the contracts path also config drive, we can get this from there and it an default to this

Reply:
Turn this into an issue? to add config for this

Reply:
Done! https://github.com/ApeWorX/ape/issues/409